src-electron/generator/matter/app/zap-templates/templates/app/helper.js

```diff
@@ -102,17 +151,20 @@ function chip_endpoint_generated_functions() {
         hasFunctionArray = true;
       }
 
-      if (configData.ClustersWithShutdownFunctions.includes(clusterName)) {
+      if (
+        configData &&
+        configData.ClustersWithShutdownFunctions.includes(clusterName)
```

Comment:
Note that there was a period when `configData` existed but `configData.ClustersWithShutdownFunctions` did not.  If we need to be compatible with SDKs from that period, we should have a check here and down below.